recipes/tbb/oneTBB/conanfile.py

```diff
@@ -0,0 +1,105 @@
+import os, shutil
+from conan.tools.cmake import CMake, CMakeToolchain
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
```

Comment:
`shutil` can be removed when applying https://github.com/conan-io/conan-center-index/pull/6887/files#r689863267
```suggestion
from conan.tools.cmake import CMake, CMakeToolchain
from conans import ConanFile, tools
from conans.errors import ConanInvalidConfiguration
import os
```